src/features/nft/addNFT/previewStep/PreviewStep.tsx

```diff
@@ -0,0 +1,146 @@
+}: TPreviewStepProps): JSX.Element {
+  const { image, onImageChange } = useChangeableImage(providedImage)
+  const [croppedImage, setCroppedImage] = useImagePreview({ image })
+  const [cropping, toggleCropping] = useToggle(false)
+
+  useEffect(() => {
+    if (croppedImage) {
+      console.log('Cropped region:', croppedImage.crop)
+    }
+  }, [croppedImage?.crop])
+
+  if (cropping && croppedImage) {
+    return (
+      <Cropping
+        image={image}
+        croppedImage={croppedImage}
+        setCroppedImage={setCroppedImage}
+        onClose={toggleCropping}
+      />
+    )
+  }
+
+  const submit = () => {
+    if (croppedImage) {
+      setCrop(croppedImage.crop)
+      goToNextStep()
+    }
+  }
+
+  return (
+    <ModalLayout
+      title='Image preview'
+      titleClass='mb-3'
+      subtitle='Description'
+      step={3}
+      leftColumnContent={
+        <div className={`relative ${style.imageContainer}`}>
+          {croppedImage && (
+            <>
+              <button
+                className={`absolute top-3.5 left-3.5 w-8 h-8 bg-gray-fc text-gray-77 flex-center rounded-full ${style.fullscreenButton}`}
+              >
+                <Expand size={13} />
+              </button>
+              <Tooltip text='Crop thumbnail'>
+                {ref => (
+                  <button
+                    ref={ref}
+                    className={`absolute bottom-3.5 left-3.5 w-10 h-10 text-white flex-center rounded-full ${style.cropButton}`}
+                    onClick={toggleCropping}
+                  >
+                    <Crop size={18} />
+                  </button>
+                )}
+              </Tooltip>
+              <label>
+                <img src={croppedImage.src} className='rounded' />
+                <input type='file' hidden onChange={onImageChange} />
+              </label>
+              {croppedImage.error && (
+                <div className='text-sm text-error font-medium mt-3'>
+                  Error text error text
+                </div>
+              )}
+            </>
+          )}
+        </div>
+      }
+      rightColumnContent={
+        <div className='space-y-8'>
+          <div>
+            <div className='font-medium text-gray-71 mb-3'>Image size</div>
+            <div className='flex'>
+              <div className='text-gray-2d text-sm font-extrabold w-20'>
+                50 Mb
+              </div>
+              <div className='flex-grow'>
+                <div className='bg-gray-e4 bg-opacity-50 rounded h-2 relative my-1'>
+                  <div
+                    className='absolute top-0 left-0 bottom-0 rounded bg-green-62'
+                    style={{ width: '65%' }}
+                  />
+                </div>
+                <div className='text-xs text-gray-71'>
+                  65% of average Pastel NFT size
+                </div>
+              </div>
+            </div>
+          </div>
```

Comment:
Can we use dummy data here? This is non-blocker.

Reply:
I saw a lot of similar comments in other PRs as well, and I'm curios, how do you think:

We will replace all data anyway during api integration. Isn't moving hardcoded value from html to a variable just a extra work?